assets/src/block-editor/Sidebar/ActionSidebar.js

```diff
@@ -34,18 +34,18 @@ export const ActionSidebar = {
             title={__('Action Options', 'planet4-blocks-backend')}>
             {isTaskTypeEnabled && (
               <SelectSidebarField
-                label={__('Task Type', 'planet4-master-theme-backend')}
+                label={__('Task Type', 'planet4-blocks-backend')}
                 options={[
                   {label: __('- Select Task Type -', 'planet4-blocks-backend'), value: 'not set'},
-                  {label: __('Do it Online', 'planet4-blocks-backend'), value: 'online'},
-                  {label: __('Do it IRL', 'planet4-blocks-backend'), value: 'irl'},
+                  {label: __('Do it Online', 'planet4-blocks'), value: 'Online'},
+                  {label: __('Do it Offline', 'planet4-blocks'), value: 'Offline'},
                 ]}
```

Comment:
You changed the values here to be capitalized but on the migration we use lowercase.

Reply:
Which way?

Reply:
Oh sorry! I missed to paste the link
https://github.com/greenpeace/planet4-master-theme/pull/2723/commits/fc383db3ddc357bd3ccd425c4875c329a1ea83c6#diff-514a704c3a75846e0d7919ec55f070a08a31a0a857461e1b64dde9bf3d8513cdL128